include/llbuild/BuildSystem/BuildSystem.h

```diff
@@ -15,6 +15,7 @@
 
 #include "llbuild/Basic/Compiler.h"
 #include "llbuild/Basic/LLVM.h"
+#include "llbuild/BuildSystem/BuildNode.h"
```

Comment:
Does this actually need to expose `BuildNode`, versus just the existing `Node`?

I don't think it does, and would rather not expose this.

Reply:
I think you're right, this is an oversight from development.